### ISSUE TITLE
`redis` Add support for `jGet()` and `jSet()`

### ISSUE THREAD

**Issue by mtuchi** [OpenFn/adaptors#709]
##3 Description 
Let's add json support like this:
```
jGet(key)
jSet(key,  json)
```

jGet and jSet (like hSet and hGet) allow json objects to be read and written.

For now, we always read/write from the root path.  Maybe later we'll add a path option.

This always replaces the object  at that key. This needs to be clearly documented.

Ref: https://github.com/OpenFn/adaptors/issues/686

**Comment by aleksa-krolls** [comment]
hey @mtuchi can we close this issue now?

**Comment by mtuchi** [comment]
@aleksa-krolls yes this is done and we have a new version now `v1.1.0`, [See docs](https://docs.openfn.org/adaptors/packages/redis-docs#jset)
